docs/reference/js/api.md

```diff
@@ -5,38 +5,129 @@ description: Developer resources for working in JavaScript with IPFS, the InterP
 
 # IPFS in JavaScript
 
-::: callout
-[Explore js-ipfs through interactive coding challenges at ProtoSchool](https://proto.school/course/ipfs)
+## JavaScript libraries
+
+There are two main JavaScript libraries for working with IPFS. Both work in Node.js and modern web browsers.
+The implementation is split into several modules. 
```

Comment:
One of the most important details to introduce here is the difference between how `js-ipfs` works in the browser and Node.js.

From a high level:
- Node.js is long-running while the browser is short-lived (and resource-constrained)
- Transport protocols: Node.js supports TCP/UDP/WebSockets while browsers are limited to HTTP, WebSockets , WebRTC. 

For a more details, check out https://www.notion.so/pl-strflt/JS-IP-2022-Roadmap-Draft-e4d3595d5e024179983a8082bc7956cf

Reply:
addressed